### PR TITLE
React & Redux group project - Space Travelers' Hub

### PR DESCRIPTION
**Pull request by Ceemos96**
### In this milestone, we: 
- Created the Space Travelers' Hub application using CRA.
- Applied React best practices and language style guides in our code
- Added components and functionality using React/Redux API.
- Performed periodic code reviews for one another.
- Fetched and rendered data to the UI.
- Ensure that there are no linter errors.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,65 @@
-# Getting Started with Create React App
+# React and Redux
```
Comment (hiromataba) [inline README.md]: - [ ] Please put the name of your project

File: README.md [hunk 2]
```diff
@@ -1,70 +1,65 @@
...
-### `npm run build`
+Click on the `Details` link to see the full output and the errors that need to be fixed:
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+![gh actions failing checks](../assets/images/gh-actions-html-css-failing-checks.png)
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+## Set-up linters in your local env
```
Comment (hiromataba) [inline README.md]: - [ ] Please update your README file by respecting the sections in the Microverse [Template](https://github.com/microverseinc/readme-template)

File: src/components/RocketCards.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { reservedRocket, cancelledReserve } from '../redux/rockets/rockets';
+import '../styles/RocketCards.css';
+
```
Comment (hiromataba) [inline src/components/RocketCards.js:6]: - [ ] Please add tests for your project as required 

image

File: src/components/profile.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
+/* eslint-disable */
```
Comment (hiromataba) [inline src/components/profile.js:1]: - [ ] Linting is a great tool so it is better to avoid disabling them to have some clean code

**Review by hiromataba — CHANGES_REQUESTED** [review]
Hi @Ceemos96 and @kelomo2502     :wave: :wave: 

almost-there

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Your PR has a proper title and description  :+1: 
- Your linters are passing :heavy_check_mark:  
- Profesional README file :rocket: 

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by algerina — APPROVED** [review]
Hi @kelomo2502 , 
Good job so far! 

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- Your project looks awesome:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous TSE


 
 ## Optional suggestions
 
 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**